src/common/config/state/init.js

```diff
@@ -89,15 +89,15 @@ const model = () => {
       },
       set block_selector (val) {
         if (isValidSelector(val)) hiddenState.block_selector += `,${val}`
-        else if (val !== '') warn('An invalid session_replay.block_selector was provided and will not be used', val)
+        else if (val !== '') warn(6, val)
       },
       // password: must always be present and true no matter what customer sets
       get mask_input_options () {
         return hiddenState.mask_input_options
       },
       set mask_input_options (val) {
         if (val && typeof val === 'object') hiddenState.mask_input_options = { ...val, password: true }
-        else warn('An invalid session_replay.mask_input_option was provided and will not be used', val)
+        else warn(7, val)
```

Comment:
I would recommend using constants that get replaced during the build step e.g. `WARN_INVALID_SESSION_REPLAY_MASK_INPUT_OPTION`

Reply:
Thank you for the suggestion @villelahdenvuo . Using constants does not reduce the bundle size so we decided to go with codes. A link will appear in the console back to the docs in the repo that can be used to lookup the error code.